examples/vic-gov-au/package.json

```diff
@@ -11,7 +11,7 @@
     "cy:open": "cypress open",
     "cy:run": "cypress run -b chrome -e TAGS='not @skip or @smoke'",
     "cy:run-record": "cypress run -b chrome -e TAGS='not @skip or @smoke' --record --parallel --group $CIRCLE_JOB",
-    "cy:run-smoke": "cypress run -e TAGS='@smoke' --spec 'test/e2e/integration/smoke/*'",
+    "cy:run-smoke": "cypress run -e TAGS='@smoke' --spec 'test/e2e/integration/smoke/*' --record false",
```

Comment:
unrelated - but needed to squash the warning about no record key used for smoke test